homeassistant/components/met/__init__.py

```diff
@@ -159,15 +151,6 @@ def set_coordinates(self) -> bool:
             longitude = self._config[CONF_LONGITUDE]
             elevation = self._config[CONF_ELEVATION]
 
```

Comment:
I meant to move the conversion to the else, i.e. when the elevation is (already) stored in the config entry, we assume the user input the elevation in feet if unit system is US.

Sorry about the misleading comment in https://github.com/home-assistant/core/pull/88283 🙈 

```suggestion
            if not self._is_metric:
                elevation = int(
                    round(
                        DistanceConverter.convert(
                            elevation, UnitOfLength.FEET, UnitOfLength.METERS
                        )
                    )
                )
```